content/en/docs/tasks/configure-pod-container/resize-container-resources.md

```diff
@@ -0,0 +1,241 @@
+---
+title: Resize CPU and Memory Resources assigned to Containers of a Pod
+content_type: task
+weight: 30
+---
+
+
+<!-- overview -->
+
+This page shows how to resize a CPU and memory resources assigned to containers
+of a running pod without restarting the pod or its containers.
+
+Kubernetes allocates resources on a node when a pod is started, and limits its
+resource use based on requests and limits specified in the pod's containers.
+
+Container's resource **requests** and **limits** are now **mutable** for CPU
+and memory resource types. They now represent **desired** CPU and memory values.
+
+This page assumes you are familiar with Quality of Service for Pods.
+
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+
+
+## Container Resize Policies
+
+Resize policies allow for a more fine-grained control over how pod's containers
+are resized for CPU and memory resource types. For example, the container's
+application may be able to handle CPU resources resized while it continues to
+run, but resizing memory limits may require that the container be restarted.
+
+To enable this, container specification allows users to specify **resizePolicy**.
+The following restart policies can be specified for resizing CPU and memory:
+* *NotRequired*: Resize the container's resources while it is running.
+* *RestartContainer*: Restart the container and apply new resources on restart.
+
+If resizePolicy.restartPolicy is not specified, it defaults to *NotRequired*.
+
+{{< note >}}
+If the pod's restartPolicy is 'Never', container's resize restart policy must be
+set to NotRequired for all containers in the pod.
+{{< /note >}}
+
+Below example shows a pod whose container's CPU can be resized without restart, but
+memory resize memory requires container to be restarted.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: qos-demo
+  namespace: qos-example
+spec:
+  containers:
+  - name: qos-demo-ctr
+    image: nginx
+    resizePolicy:
+    - resourceName: cpu
+      restartPolicy: NotRequired
+    - resourceName: memory
+      restartPolicy: RestartContainer
+    resources:
+      limits:
+        memory: "200Mi"
+        cpu: "700m"
+      requests:
+        memory: "200Mi"
+        cpu: "700m"
+```
+
+{{< note >}}
+In the above example, if desired resources for both CPU and memory have changed,
```

Comment:
Please be specific when talking about "desired resources" ... Are you referring to `requests` or `limits` or both?